{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2022-43903",
    "ASSIGNER": "psirt@us.ibm.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "NVD-CWE-noinfo"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://www.ibm.com/support/pages/node/7030110",
        "name": "https://www.ibm.com/support/pages/node/7030110",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240894",
        "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240894",
        "refsource": "MISC",
        "tags": [
          "VDB Entry",
          "Vendor Advisory"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "IBM Security Guardium 10.6, 11.3, and 11.4 could allow an authenticated user to cause a denial of service due to due to improper input validation.  IBM X-Force ID:  240894."
      }
    ]
  }
}